### PR TITLE
Fan Relto video to replace Yeesha Intro

### PR DESCRIPTION
**Pull request by Hazado**
Data files - https://github.com/H-uru/moul-assets/pull/218

### CODE REVIEW

File: Scripts/Python/xOpeningSequence.py [hunk 1]
```diff
@@ -75,7 +75,7 @@
 
 gIntroMovie = None
 kAtrusIntroMovie = "avi/AtrusIntro.webm"
-kYeeshaIntroMovie = "avi/URULiveIntro.webm"
+kIntroMovie = "avi/NewPlayerIntro.webm"
```
Comment (dpogue) [inline Scripts/Python/xOpeningSequence.py:78]: Having variables named `gIntroMovie` and `kIntroMovie` (particularly when `kIntroMovie` is not entirely a constant) seems prone to confusion...

Maybe something like `gMovieFilePath` makes more sense as the variable, and have `kAtrusIntroMovie` and `kReltoIntroMovie` as string constants that get assigned to that filepath variable (along with the vault override)?

Comment (Hazado) [reply Scripts/Python/xOpeningSequence.py]: Done

**Review by dpogue — APPROVED** [review]
LGTM

**Comment by Hazado** [comment]
So I needed to delay the SDL check for setting the video for 3 seconds.
Without the delay it was getting the SDL but it would have the default values instead of what was set on the server.
With a delay in there it gets the correct server SDL

**Comment by dpogue** [comment]
Rather than a hardcoded delay, can you use `OnServerInitComplete` or something? There should be a Python callback for when the server has sent the initial SDL state...

File: Scripts/Python/xOpeningSequence.py [hunk 2]
```diff
@@ -223,10 +225,14 @@ def OnGUINotify(self,id,control,event):
                 # see if the there actually is a movie to play
                 skipMovie = 1
                 try:
+                    ageSDL = PtGetAgeSDL()
                     if IsTutorialPath():
-                        os.stat(kAtrusIntroMovie)
+                        gMovieFilePath = kAtrusIntroMovie
+                    elif ageSDL["psnlIntroMovie"]:
```
Comment (Hoikas) [inline Scripts/Python/xOpeningSequence.py:231]:
```suggestion
                    elif ageSDL["psnlIntroMovie"][0]:
```
The current code works, but it's somewhat misleading. What we're really testing is the actual value of the variable being non-empty.

File: Scripts/Python/xOpeningSequence.py [hunk 3]
```diff
@@ -223,10 +225,14 @@ def OnGUINotify(self,id,control,event):
...
+                        gMovieFilePath = ageSDL["psnlIntroMovie"][0]
                     else:
-                        os.stat(kYeeshaIntroMovie)
+                        gMovieFilePath = kReltoIntroMovie
+                    os.stat(gMovieFilePath)
```
Comment (Hoikas) [inline Scripts/Python/xOpeningSequence.py:235]:
```suggestion
```

This doesn't do anything useful.

File: Scripts/Python/xOpeningSequence.py [hunk 4]
```diff
@@ -328,10 +334,14 @@ def OnGUINotify(self,id,control,event):
         elif id == -1:
             if event == kShowHide:
                 if control.isEnabled():
+                    ageSDL = PtGetAgeSDL()
                     if StartInCleft():
-                        gIntroMovie = ptMoviePlayer(kAtrusIntroMovie, self.key)
+                        gMovieFilePath = kAtrusIntroMovie
+                    elif ageSDL["psnlIntroMovie"]:
```
Comment (Hoikas) [inline Scripts/Python/xOpeningSequence.py:340]:
```suggestion
                    elif ageSDL["psnlIntroMovie"][0]:
```

File: Scripts/Python/xOpeningSequence.py (hunk 3 above)
Comment (dpogue) [reply Scripts/Python/xOpeningSequence.py:235]: Isn't the intention here to check if the file exists, and go into the `except` block if it's not found?

Comment (Hoikas) [reply Scripts/Python/xOpeningSequence.py:235]: Hmm... The `except` block isn't visible in the context lines. Ok, carry on.

**Comment by Hazado** [comment]
Consolidated a bit of the code for the movie check
It will now properly fail and skip the movie GUI if the movie is not found

Added your requested changes Adam

**Pull request merged**